lib/tokens/session_token.js

```diff
@@ -16,105 +15,114 @@ const TOKEN_FRESHNESS_THRESHOLD = 50 * 365 * 24 * ONE_HOUR // 50 years or post Y
 module.exports = (log, Token, config) => {
   const features = require('../features')(config)
 
-  function SessionToken(keys, details) {
-    Token.call(this, keys, details)
-    this.setUserAgentInfo(details)
-    this.setDeviceInfo(details)
-    this.email = details.email || null
-    this.emailCode = details.emailCode || null
-    this.emailVerified = !! details.emailVerified
-    this.verifierSetAt = details.verifierSetAt
-    this.locale = details.locale || null
-    this.mustVerify = !! details.mustVerify || false
-
-    // Tokens are considered verified if no tokenVerificationId exists
-    this.tokenVerificationId = details.tokenVerificationId || null
-    this.tokenVerified = this.tokenVerificationId ? false : true
-  }
-  inherits(SessionToken, Token)
-
-  SessionToken.tokenTypeID = 'sessionToken'
-
-  SessionToken.create = function (details) {
-    details = details || {}
-    log.trace({ op: 'SessionToken.create', uid: details.uid })
-    return Token.createNewToken(SessionToken, details)
-  }
+  class SessionToken extends Token {
```

Comment:
Gotta be honest, I panicked a little when I saw changes to this file given the recent issues with reset token timestamps.  But it looks like a straightforward syntactic shuffle so 👍